shiro-gg.js.org

moyshik7 opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

moyshik7 commented:
The `CNAME` file is in `/docs` folder because that's where the docs are.

MattIPv4 commented on cnames_active.js:
```diff
@@ -2297,6 +2297,7 @@ var cnames_active = {
   "shikwasa": "jessuni.github.io/shikwasa",
   "shine": "shinenelson.github.io/shine.js.org",
   "shirajuki": "shirajuki.github.io",
+  "shiro.gg": "moyshik7.github.io/shiro.gg",
```
```suggestion
  "shiro.gg": "moyshik7.github.io/shiro.gg", // noCF
```

moyshik7 replied:
my GitHub repo and npm package is named `shiro.gg` should I apply for `shiro-gg` instead ?  
So that it doesn't turn into a sub sub domain

indus replied:
I would encourage "shiro-gg" and the usage of Universal SSL of Cloudflare. But its up to you. In' case you go with ```shiro-gg``` you should remove the ````// noCF```

moyshik7 replied:
I've changed the name to shiro-gg.  
Thanks for the suggestion  
💜💜💜

MattIPv4 reviewed (changes requested):
👋 As you are requesting a sub-sub-domain, the Cloudflare proxy and automatic SSL will not be available, so you will need to add the `// noCF` flag to your request.

moyshik7 commented:
my GitHub repo and npm package is named `shiro.gg` can I apply for `shiro-gg` instead ?  
So that it doesn't turn into a sub sub domain

moyshik7 commented:
Okay I changed the name of my GitHub repo to `shiro-gg`  
and changed the name in the file as well.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to clean up your ```CNAME``` file to make it work. Just a single line with: shiro-gg.js.org
I'll merge after we have "contact".

moyshik7 commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to clean up your `CNAME` file to make it work. Just a single line with: shiro-gg.js.org
> I'll merge after we have "contact".

Thanks The error was possibly because of thedocs generator I used and it overlapped the CNAME.  
Fixed it now.  
Thanks

indus commented:
Welcome https://shiro-gg.js.org
